extensions/sftp/tests/FetchSFTPTests.cpp

```diff
@@ -75,10 +73,10 @@ class FetchSFTPTestsFixture {
     LogTestController::getInstance().setDebug<SFTPTestServer>();
 
     // Create temporary directories
-    testController.createTempDirectory(src_dir);
-    REQUIRE(src_dir != nullptr);
-    testController.createTempDirectory(dst_dir);
-    REQUIRE(dst_dir != nullptr);
+    src_dir = testController.createTempDirectory();
+    REQUIRE_FALSE(src_dir.empty());
+    dst_dir = testController.createTempDirectory();
+    REQUIRE_FALSE(dst_dir.empty());
```

Comment:
Did you consider moving the initializers to the member initializer list or to default member initializers?

Reply:
I did not consider it, but it is a good idea, thanks!  Done in https://github.com/apache/nifi-minifi-cpp/pull/1139/commits/555d7b367f11d7750d594f2c0076b1e4c4144cdd.